src/DAP_config.h

```diff
@@ -183,6 +183,16 @@ Provides definitions about:
 #define PIN_LED_CONNECTED PIN_LED_TXL
 #define PIN_LED_RUNNING   PIN_LED_RXL
 
+#else
```

Comment:
I am not sure which should be best here, i think A0, A1, A2 is most common appeared in all the boards. Of course special board with special need have to modify this. I am just opt to make it easier to work without any modification. @ladyada do you have any suggestion rather than A0, A1, A2.